extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegesBuilder.scala

```diff
@@ -83,6 +84,27 @@ object PrivilegesBuilder {
     spark.sessionState.catalog.isTempView(parts)
   }
 
+  private def isPersistentFunction(
+      functionIdent: FunctionIdentifier,
+      spark: SparkSession): Boolean = {
+    val (db, funcName) = functionIdent.database match {
+      case Some(db) =>
+        (db, functionIdent.funcName)
+      case _ =>
+        Try {
+          spark.sessionState.catalog.lookupFunctionInfo(functionIdent)
```

Comment:
Is this `lookupFunctionInfo` necessary for function identifier with null dbs,
And in what case it will be will out with funInfo?

Reply:
> Is this `lookupFunctionInfo` necessary for function identifier with null dbs, And in what case it will be will out with funInfo?

In query 'DESCRIBE FUNCTION fun1', DescribeFunctionCommand's functionName field is `FunctionIdentifier(funName, None)`, even the fun1 is a persistent function, unless you use the full name: 'DESCRIBE FUNCTION db.fun1'.

So, it's do necessary in #isPersistentFunction method.

Reply:
`lookupFunctionInfo` is a heavy action after all.
And I think we have resolved functions and check the privileges only for here.
So even if db name is missing, let's just fill it with current db name , just like in `spark.sessionState.catalog.lookupFunctionInfo`.

Shall try some ut without `lookupFunctionInfo` to verify its necessity？

Reply:
sounds reasonable, when db is missing, we use current db to check wheather it's persistent. 
`spark.sessionState.catalog.isPersistentFunction(FunctionIdentifier(funcName, database.or(spark.catalog.currentDatabase)))`

Reply:
Let's just skip filling the missing db name.
As in `spark.sessionState.catalog.isPersistentFunction`, it will be auto filled with current db name.

```
def isPersistentFunction(name: FunctionIdentifier): Boolean = {
    val db = formatDatabaseName(name.database.getOrElse(getCurrentDatabase))
...
```